var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -32,9 +32,6 @@ class Magma(CMakePackage):
     depends_on('lapack')
     depends_on('cuda')
```

Comment:
Do we need to get rid of this depends_on? Can `magma` be build with or without `cuda`?

Reply:
I was wondering about that too.

Reply:
it seems we can get rid of `depends_on('cuda..` line however it seems cuda is a requirement for magma as installation instructions strongly imply so also a comment made in this link: https://icl.cs.utk.edu/magma/forum/viewtopic.php?f=2&t=2336